projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/AsPath.java

```diff
@@ -15,6 +15,10 @@
 
   private static final long serialVersionUID = 1L;
 
+  public static boolean isPrivateAs(int as) {
+    return (as >= 64512 && as <= 65535);
```

Comment:
Please add a comment with a link to the RFC. My javadoc proposal:

```java
/**
 * Returns true if the provided AS number is reserved for private use by RFC 6696:
 * https://tools.ietf.org/html/rfc6996#section-5
 */
```

My reasoning: makes it self-evident why this goes in data model (is vendor independent) and is educational for new people.